anvilPoll2024ExtraAnalysis.Rmd

```diff
@@ -869,7 +869,21 @@ ggsave(here("plots/stackedbarplot_rankfeatures.png"))
 
 ## Preferences: DMS compliance/data repositories (supplemental)
 
-*No supplements at this time*
+How many repositories did users select/report?
+
+```{r}
+numReposPlot <- resultsTidy %>% 
+  mutate(numRepos = (str_count(RepositoriesDMS, ",|&")+1)) %>% #count commas or an and sign and add one since 2 commas means three responses
```

Comment:
Extra parentheses here? Does the "&" appear with two selections?

Reply:
I think the extra parentheses are because I'm adding one to the str_count results. I'm not sure how many selections have a `&`. I think it's just one.

Reply:
Oh, sorry, I think I understand your question now. Yes, there's an other response where someone used an `&` to separate two different choices

Reply:
re: parens, you should be able to do `mutate(numRepos = str_count(RepositoriesDMS, ",|&")+1)`